src/Symfony/Component/AssetMapper/ImportMap/ImportMapRenderer.php

```diff
@@ -114,21 +114,26 @@ public function render(string|array $entryPoint, array $attributes = []): string
         }
 
         if ($polyfillPath) {
-            $url = $this->escapeAttributeValue($polyfillPath);
-            $polyfillAttributes = $scriptAttributes;
+            $polyfillAttributes = $attributes + $this->scriptAttributes;
 
             // Add security attributes for the default polyfill hosted on jspm.io
             if (self::DEFAULT_ES_MODULE_SHIMS_POLYFILL_URL === $polyfillPath) {
-                $polyfillAttributes = $this->createAttributesString([
+                $polyfillAttributes = [
                     'crossorigin' => 'anonymous',
                     'integrity' => self::DEFAULT_ES_MODULE_SHIMS_POLYFILL_INTEGRITY,
-                ] + $attributes);
+                ] + $polyfillAttributes;
             }
 
             $output .= <<<HTML
-
-                <!-- ES Module Shims: Import maps polyfill for modules browsers without import maps support -->
-                <script async src="$url"$polyfillAttributes></script>
+                <script async$scriptAttributes>
+                if (!HTMLScriptElement.supports || !HTMLScriptElement.supports('importmap')) (function () {
```

Comment:
I confirmed this works using Browerstack :shipit:

Reply:
Soon in Nicolas' bio "GPT+BrowserStack powered JS dev" 😁